### PR TITLE
DOCS: add SMC back

### LINKED ISSUE #3232: pm.Model() is missing in Bayes_factor.ipynb
In the notebook Bayes_factor.ipynb the initialization of pm.Model() cell is missing, please do resolve it.

### PR DESCRIPTION
**Pull request by aloctavodia**
fix #3232

### CODE REVIEW

**Comment by aloctavodia** [comment]
@twiecki let me know if you have any suggestion.

File: docs/source/notebooks/Bayes_factor.ipynb [hunk 1]
```diff
@@ -273,6 +273,84 @@
     "We see that the model with the more concentrated prior $Beta(30, 30)$ has $\\approx 5$ times more support than the model with the more extended prior $Beta(1, 1)$. Besides the exact numerical value this should not be surprising since the prior for the most favoured model is concentrated around $\\theta = 0.5$ and the data $y$ has equal number of head and tails, consintent with a value of $\\theta$ around 0.5."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sequential Monte Carlo\n",
+    "\n",
+    "The [Sequential Monte Carlo](SMC2_gaussians.ipynb) sampler is a method that basically progress by a series of successive interpolated (or _annealed_) sequences from the prior to the posterior. A nice _by product_ of this process is that we get an estimation of the marginal likelihood."
```
Comment (twiecki) [inline docs/source/notebooks/Bayes_factor.ipynb]:
```suggestion
    "The [Sequential Monte Carlo](SMC2_gaussians.ipynb) sampler is a method that basically progresses by a series of successive interpolated (or _annealed_) sequences from the prior to the posterior. A nice by-product of this process is that we get an estimation of the marginal likelihood."
```

File: docs/source/notebooks/Bayes_factor.ipynb [hunk 2]
```diff
@@ -273,6 +273,84 @@
...
+   "source": [
+    "BF_smc = models[1].marginal_likelihood / models[0].marginal_likelihood\n",
+    "print(round(BF_smc))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see from the previous cell, SMC give essentially the same answer than the analytical calculation! \n",
```
Comment (twiecki) [inline docs/source/notebooks/Bayes_factor.ipynb]:
```suggestion
    "As we can see from the previous cell, SMC gives essentially the same answer as the analytical calculation! \n",
```

File: docs/source/notebooks/Bayes_factor.ipynb [hunk 3]
```diff
@@ -273,6 +273,84 @@
...
+    "\n",
+    "The advantage of using SMC is that we can use it to compute the _marginal likelihood_ for a wider range of models, given that we do not need an analytical expression for the _marginal likelihood_. The cost we pay is that the computations is more expensive with SMC, we should take into account that for more complex models we should need to increase the number of `n_steps` and `n_chains` for a more accurate estimation of the _marginal likelihood_."
```
Comment (twiecki) [inline docs/source/notebooks/Bayes_factor.ipynb]:
```suggestion
    "The advantage of using SMC is that we can use it to compute the _marginal likelihood_ for a wider range of models, given that we do not need an analytical expression for the _marginal likelihood_. The cost we pay is that the computation is more expensive with SMC. We should also take into account that for more complex models we should increase the number of `n_steps` and `n_chains` for a more accurate estimation of the _marginal likelihood_."
```

**Comment by twiecki** [comment]
Looks great. Kind of striking that the model is 5x better but would provide the same accuracy, which is probably what we really care about, thus a case for using e.g. LOO-WAIC instead. Maybe you could even add that point.

**Comment by twiecki** [comment]
Thanks @aloctavodia !

**Pull request merged**